crates/proto/src/xfer/dnssec_dns_handle/nsec3_validation.rs

```diff
@@ -1,3 +1,72 @@
+//! Throughout this module several NSEC3 specific terms are used.
+//!
+//! "closest_encloser" - a name that is a parent / grandparent /
```

Comment:
What would you say about using "ancestor" to encompass parent, grandparent, etc.

I think it is also important to clarify that the closest encloser is the longer ancestor that exists in the zone and not just any ancestor that exists in the zone.